packages/orm/src/getProperties.ts

```diff
@@ -44,57 +46,56 @@ export function getProperties(meta: EntityMetadata): Record<string, any> {
   // the GraphQL resolvers. So we should probably just remove this filter and let everything
   // get returned as properties.
   const knownPrimitives = Object.values(meta.allFields)
-    .filter((f) => f.kind === "primaryKey" || f.kind === "primitive" || f.kind === "enum")
+    .filter((f) => f.kind === "primaryKey" || ((f.kind === "primitive" || f.kind === "enum") && !f.derived))
     .map((f) => f.fieldName);
 
-  const properties = Object.fromEntries(
-    // Recursively looking for ownKeys will find:
-    // - Custom properties set on the instance, like `readonly author: Reference<Author> = hasOneThrough(...)`
-    // - Getters declared within the class like `get initials()`
-    // - Getters auto-created by transform-properties when it lazies `readonly author = hasOneThrough(...)` relations
-    // - Getters declared within the codegen classes like `get books(): Reference<...>`
-    getRecursiveOwnNames(instance)
-      .filter((key) => key !== "constructor" && !key.startsWith("__") && !knownPrimitives.includes(key))
-      .map((key) => {
-        // Return the value of `instance[key]` but wrap it in a try/catch in case it's
-        // a getter that runs code that fails b/c of the dummy state we're in.
-        try {
-          return [key, (instance as any)[key] ?? unknown];
-        } catch {
-          return [key, unknown];
-        }
-      })
-      // Purposefully return methods, primitives, etc. so that `entityResolver` can add them to the resolver
-      .filter(([key]) => key !== "fullNonReactiveAccess" && key !== "transientFields"),
```

Comment:
We temporarily return `transientFields`, so that we can getter-ize it, but otherwise later in this method re-filter it out to keep the same API contract